src/.editorconfig

```diff
@@ -297,6 +297,7 @@ csharp_style_prefer_local_over_anonymous_function = true:suggestion
 dotnet_diagnostic.SA1602.severity = suggestion
 dotnet_diagnostic.SA1307.severity = silent
 csharp_style_implicit_object_creation_when_type_is_apparent = true:suggestion
+csharp_style_prefer_primary_constructors = true:suggestion
```

Comment:
>true [](http://example.com/codeflow?start=43&length=4)

interestingly, this item was added when I explicitly used 'dotnet format' to get all autofixes I could from the command-line. This must be a tool feature, to add new diagnostics as they are added to the toolchain.